lib/config-loader.js

```diff
@@ -21,6 +21,10 @@ const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-app-scripts
 const defaultAioHostname = 'adobeio-static.net'
 const defaultTvmUrl = 'https://adobeio.adobeioruntime.net/apis/tvm/'
 const defaultOwApiHost = 'https://adobeioruntime.net'
+const defaultHTMLCacheDuration = '60'
+const defaultJSCacheDuration = '604800'
+const defaultCSSCacheDuration = '604800'
+const defaultImageCacheDuration = '604800'
```

Comment:
should we consider all resources instead of just images? In which case we would rename to resourceCacheDuration and match else im getCacheDuration for all other mime types
wdyt?

Reply:
Images might be a very common category of resources for which one might want to control the caching. 
Maybe we can have an extra resourceCacheDuration for all other resources?

Reply:
ok yes makes sense, agree just the naming then might be confusing as an image is a resource? not sure

Reply:
On second thought, let's keep the name as is. 
We are not sure what other non image resources might get used, so it's better not to add any cache control for them from our side.

Reply:
Ok LGTM